components/tools/OmeroPy/test/integration/scriptstest/test_util_scripts.py

```diff
@@ -162,3 +164,80 @@ def test_dataset_to_plate(self):
                 count += 1
 
         assert count == n
+
+    @pytest.mark.parametrize("remove", [True, False])
+    def test_move_annotations(self, remove):
+
+        script_id = super(TestUtilScripts, self).get_script(move_annotations)
+        assert script_id > 0
+
+        # session = self.root.sf
+        client = self.root
+        field_count = 2
+
+        plate = self.import_plates(client=client, fields=field_count)[0]
+
+        def link_ann(client, ann, image_id):
```

Comment:
you can use the ``link`` method from ``testlib`` instead of introduction a new method in the test
If need be we can add more type